test/general/TestLocations.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+from collections import Counter
+from worlds.AutoWorld import AutoWorldRegister
+from . import setup_default_world
+
+
+class TestBase(unittest.TestCase):
+    def testCreateDuplicateLocations(self):
+        for game_name, world_type in AutoWorldRegister.world_types.items():
+            if game_name in {"Final Fantasy"}:
+                continue
+            multiworld = setup_default_world(world_type)
+            locations = Counter([location for location in multiworld.get_locations()])
```

Comment:
```suggestion
            locations = Counter(multiworld.get_locations())
```
Writing this suggestion again after it was invalidated by the last commit

Reply:
🤦